llvm/utils/TableGen/ARMTargetDefEmitter.cpp

```diff
@@ -53,12 +53,12 @@ static void CheckFeatureTree(const Record *Root) {
   }
 }
 
-static void EmitARMTargetDef(const RecordKeeper &RK, raw_ostream &OS) {
+static void emitArmTargetDef(const RecordKeeper &RK, raw_ostream &OS) {
```

Comment:
Probably should keep ARM capitalized here

Reply:
fixed in most recent commit